tests/Unit/Extensions/ImagesExtensionTest.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace Xefi\Faker\Images\Tests\Unit\Extensions;
+
+use Intervention\Image\Image;
+use Random\Randomizer;
+use Xefi\Faker\Container\Container;
+use Xefi\Faker\Images\Exceptions\NoDriverException;
+use Xefi\Faker\Images\Tests\Unit\TestCase;
+
+final class ImagesExtensionTest extends TestCase
+{
+    public function testImage(): void
+    {
+        $faker = new Container(false);
+
+        if (!extension_loaded('gd') && !extension_loaded('imagick')) {
+            $this->expectException(NoDriverException::class);
+        }
+
+        $image = $faker->unique()->image();
+
+        if (extension_loaded('gd') || extension_loaded('imagick')) {
+            $this->assertInstanceOf(Image::class, $image);
+            $this->assertEquals(300, $image->width());
+            $this->assertEquals(200, $image->height());
+        }
```

Comment:
In my opinion you should have two tests, both skips when the condition isn't good ($this->markAsSkipped())

So the test name is more clear

Reply:
Ok I made different tests for each extension